Common/cpp/Tools/JSISerializer.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <jsi/jsi.h>
+#include <sstream>
+#include <string>
+
+using namespace facebook;
+
+namespace reanimated {
+
+std::string stringifyValue(jsi::Runtime &rt, const jsi::Value &value);
+std::string stringifyValueRecursively(
+    jsi::Runtime &rt,
+    const jsi::Value &value);
+std::string stringifyJSIArray(jsi::Runtime &rt, const jsi::Array &arr);
+std::string stringifyJSIArrayBuffer(
+    jsi::Runtime &rt,
+    const jsi::ArrayBuffer &buf);
+std::string stringifyJSIFunction(jsi::Runtime &rt, const jsi::Function &func);
+std::string stringifyJSIHostObject(
+    jsi::Runtime &rt,
+    jsi::HostObject &hostObject);
```

Comment:
```suggestion
    const jsi::HostObject &hostObject);
```

Reply:
HostObject method `getPropertyNames` is not marked `const`, so i can't use it actually.